src/modules/client/client.controller.ts

```diff
@@ -0,0 +1,75 @@
+import { ClientService } from './client.service';
+import {
+  Controller,
+  Get,
+  HttpCode,
+  UseGuards,
+  Param,
+  Body,
+  Post,
+} from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { CandidateFilterDto } from './dto/candidate-filter.dto';
```

Comment:
Body and CandidateFilterDto aren't used in this file.

Reply:
+